### ISSUE TITLE
Incorrect depdendency versions?

### ISSUE THREAD

**Issue by audunhalland** [pact-foundation/pact-reference#48]
Hi, I now have a new experiment where I'm writing a consumer in Rust. Adding to my test dependencies

```
[dev-dependencies]
pact_consumer = "0.5.1"
```

(which now is the latest published version) and compiling this, I get a compile error, because versions do not seem to be matching here:

```
   --> /Users/audun/.cargo/registry/src/github.com-1ecc6299db9ec823/pact_mock_server-0.5.1/src/hyper_server.rs:141:28
    |
141 |             let response = pact_matching::generate_response(&interaction.response);
    |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 2 parameters

```

So it seems that `cb30a2f2 feat: added the ProviderStateGenerator as a generator type` has been released, but `pact_consumer 0.5.1` depends on a newer version of `pact_matching` than it really supports (it depends on `pact_matching 0.5.3`).

I'm just guessing that the release process involves a good deal of manual version increments, and seems easy to get wrong. I've never worked with releases from Rust monorepos myself, so I don't really have a clue how the process can be improved.

**Comment by uglyog** [comment]
Oh, damn. I regret making that change on master. It should have been in a branch.

I'll release an updated crate for you. But I agree, we need to find a way to handle changes like this.

**Comment by uglyog** [comment]
I've released 0.5.2
